model/src/test_manager/status.rs

```diff
@@ -86,6 +102,8 @@ impl std::fmt::Display for StatusSnapshot {
 
 impl From<&StatusSnapshot> for Table {
     fn from(status: &StatusSnapshot) -> Self {
+        let mut crds = status.crds.clone();
+        crds.sort_by_key(|crd| crd.name());
```

Comment:
With the additional complexity, some unit tests might be nice to have for this trait implementation.

Reply:
I'm not too familiar with `tabled::Table` so maybe it's not easy to `asset_eq` to ensure the generated table is as expected in a unit test.